internals/models.py

```diff
@@ -786,7 +787,9 @@ def get_all(self, limit=None, order='-updated', filterby=None,
           # This includes being an owner, editor, or the original creator
           # of the feature.
           query = query.filter(
-            ndb.OR(Feature.owner == comparator, Feature.editors == comparator))
+            ndb.OR(Feature.owner == comparator,
+                   ndb.OR(Feature.editors == comparator,
```

Comment:
FYI, this might require an update to index.yaml.  ISTR, the docs say that we need an index that exactly matches the set of fields used in each query (including sorting).  However, I have a feeling that an index is needed for each OR clause rather than for the overall query.  

Single field queries use an implicitly created single-field index.  So, if it is by OR clause, we might not need any explicit index.  I am thinking that one of these OR clauses gets combined with the ordering, so that is why we needed to add that one index last time.

  Let's test it on staging and see if we get a helpful error message.

Reply:
Okay sounds good. I'm not familiar with finding correct indexes, so I'd be interested in knowing your process for index troubleshooting.